app/styles/module/article/_section-header.scss

```diff
@@ -7,6 +8,42 @@ h2[section] {
       display: block;
     }
   }
+
+  .chevron-wrapper {
+    align-items: center;
+    display: flex;
+    height: 44px;
+    margin-right: -$article-horizontal-padding;
+    padding: 9px 13px;
+
+    .chevron {
+      fill: $wds-fandom-color-middle-gray;
+    }
+  }
+
+  .vertical-separator {
+    border-left: 1px solid $wds-fandom-color-middle-gray;
+    height: 26px;
+    width: 1px;
+  }
+
+  .section-edit-link {
+    align-items: center;
+    color: $wds-fandom-color-middle-gray;
+    display: flex;
+    height: 44px;
+    padding: 9px 13px;
+  }
+}
+
+.hide-edit-links {
+  h1[section],
+  h2[section] {
+    .section-edit-link,
```

Comment:
could be
```
@at-root .hide-edit-links & {}
```
in the respective blocks above

Reply:
I'm not scss wizzard but @at-root makes code less readable to me